test/DebuggerCommon/ES6_proto_invalidation.js.dbg.baseline

```diff
@@ -459,13 +465,6 @@
               "caller": "object null",
               "arguments": "object null"
             },
-            "indexOf": {
-              "#__proto__": "function <large string>",
-              "length": "number 1",
-              "name": "string indexOf",
-              "caller": "object null",
-              "arguments": "object null"

```

Comment:
we need to understand why we no longer have `caller` or `arguments` and we have new `prototype` property.